sormas-api/src/main/resources/strings.properties

```diff
@@ -521,15 +524,15 @@ infoUploadDocumentTemplate=Select a ".docx"-Document Template you would like to
-infoCaseCompleteness = Completeness is a measurement of how much vital data has already been entered into the case. This is composed of the following, weighted by their importance:<br/><ul><li>Has the case been classified?</li><li>Has the case been investigated?</li><li>Has at least one sample been taken?</li><li>Has at least one symptom been documented?</li><li>Has at least one contact been created?</li><li>Is there an outcome for the case?</li><li>Does the case have a birth date or age?</li><li>Does the case have a sex?</li><li>Has an onset date been specified?</li></ul>
-infoCompletenessMerge = You can use this measurement to get an idea about which case you should use as the one that remains active when merging. However, please make sure that you still have a detailed look at both cases to avoid mistakes.
-infoMergeIgnoreRegion = Selecting this option excludes the region as a criteria for detecting duplicates. Merging and picking are deactivated here because the probability of a real duplicate beyond region boundaries is very low. If a real duplicate is detected, this case can be deleted manually via the case directory.
+infoMergingExplanation = This view is designed to assist you in detecting and merging duplicate entities. entity are always displayed as pairs, the entity with the earlier creation date being on top.<br/><br/>You can use the creation date filter to only view potential duplicates that were created in a specific time period. The creation date is automatically generated when a entity is entered into SORMAS and can not be changed by users. Therefore, once you have reviewed all duplicates from a specific time period in the past, you will never have to do it again as there will be no new entities created during this time frame.<br/><br/>Please note that most likely not all displayed entities are actual duplicates. There is the potential for false positives, so <b>please thoroughly review the entities before merging them!</b> In addition to the information provided in the table, you can also click on the IDs of the entities to open them in a new tab. Keep in mind that it is much worse to falsely merge unique entities together than to have some duplicates in the system.
+infoMergingMergeDescription = Choose this option only if you are <b>sure that the two cases/contacts are the same!</b> You should click on this button next to the entity with the most recent or more complete information. Click on the IDs of the entities to open them in new tabs in order to view all their data. When the entity you chose is missing information that is present in the other entity, it will be updated. Also, contacts, samples, tasks and case management information will be transfered. Afterwards, the case you did not choose will be deleted.
```

Comment:
You probably have to use separate strings for "infoMergingPickDescription" for contacts and cases because this string refers to contacts being transfered, which obviously isn't the case when contacts are merged.

Reply:
Are you speaking about "infoMergingPickDescription" or "infoMergingExplanation" ? If it is about "infoMergingPickDescription", it is only used once in MergeGuideLayout for cases and contacts. Can we call different Strings from the same Label depending on the imports.

Reply:
No, there's a passage in "infoMergingPickDescription" that refers to contacts being transfered, which doesn't really make sense if you're merging contacts.

Reply:
infoMergingPickDescription = Clicking on this button will simply delete the entity that was not picked. However, no information will be copied to the entity that remains active. All samples and tasks will be lost as well. **Choose this option if you are sure that both entities are the same, but the other entity has not been updated with any information that needs to be transfered to the one that remains active.**    

Are you talking about this last part of the sentence ?

Do we need to have different sentences for cases and for contacts merges ?  This sentence is used once in MergeGuideLayout for cases and contacts. Can we call different Strings from the same Label depending on the imports in MergeGuideLayout ?

Reply:
Oh, I'm sorry! I referenced the wrong property, that's my bad. Have a look at "infoMergingMergeDescription" please, that string includes the passage that I was talking about.